templates/control

```diff
@@ -29,23 +29,36 @@ Depends: apache2 | apache | apache-ssl | apache-perl | httpd,
     logrotate,
     mysql-client,
     mysql-server,
+    libjs-jquery-timepicker,
+    libjs-bootstrap,
+    libjs-jquery-cookie,
+    libjs-jquery-ui,
+    libjs-jquery-ui-theme-ui-lightness,
```

Comment:
we have a custom jquery-ui theme, not the default lightness theme.
See https://github.com/glpi-project/glpi/blob/master/lib/jquery/css/smoothness/jquery-ui-1.10.4.custom.css